### DISCUSSION TITLE
[Projects Beta] Allow project deletion

### DISCUSSION THREAD

**Discussion by HeCorr** [community/community#16822] [Projects and Issues] [answered]
Hello! At the moment it seems you can only _close_ (beta) projects but it would be great if I could completely _delete_ them.

Thanks!

**Comment by 2percentsilk** [comment, marked as answer]
We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need). Give it a try and let us know how it goes.

**Reply by HeCorr** [reply]
It's a bit hidden but works just fine. Thank you!
